src/main/java/org/example/homework9/MobilePhone.java

```diff
@@ -0,0 +1,71 @@
+package org.example.homework9;
+import java.util.ArrayList;
+
+public class MobilePhone {
+    private final ArrayList<Contact> myContacts;
+
+    public MobilePhone(String myNumber) {
+        this.myContacts = new ArrayList<>();
+    }
```

Comment:
empty line is needed

Reply:
I added changes to the code based on your comments